backend/data/skills.yaml

```diff
@@ -241,16 +241,96 @@ requirements:
 
   _logistics: &logistics
     # Logistics skills: all logi pilots need these
-    Advanced Target Management:
+    EM Armor Compensation:
+      min: 4
+      elite: 5
+    Explosive Armor Compensation:
+      min: 4
+      elite: 5
+    Thermal Armor Compensation:
+      min: 4
+      elite: 5
+    Kinetic Armor Compensation:
+      min: 4
+      elite: 5
+    Armor Layering:
+      elite: 4
+    Hull Upgrades:
+      min: 5
+      elite: 5
+    Armor Rigging:
+      elite: 4
+    Mechanics:
+      min: 4
+      elite: 5
+    Acceleration Control:
       min: 3
+      elite: 4
+    Evasive Maneuvering:
+      elite: 4
+    Warp Drive Operation:
+      min: 3
+      elite: 5
+    Navigation:
+      min: 4
+      elite: 5
+    Drones:
+      min: 3
+      elite: 5
+    Gallente Drone Specialization:
+      elite: 4
+    Drone Durability:
+      elite: 4
+    Drone Navigation:
+      elite: 4
+    Drone Avionics:
+      elite: 4
+    Advanced Drone Avionics:
+      elite: 4
+    Drone Interfacing:
+      elite: 4
+    Drone Sharpshooting:
+      elite: 4
+    Thermodynamics:
+      elite: 4
     Capacitor Management:
       min: 5
+      elite: 5
+    CPU Management:
+      min: 3
+      elite: 5
+    Remote Armor Repair Systems:
+      min: 5
+      elite: 5
     Capacitor Systems Operation:
       min: 5
-    Acceleration Control:
+      elite: 5
+    Power Grid Management:
       min: 3
-    Hull Upgrades:
+      elite: 5
+    Spaceship Command:
+      elite: 5
+    Nanite Interfacing:
+      elite: 4
+    Nanite Operation:
+      elite: 4
+    Target Management:
       min: 5
+      elite: 5
+    Long Range Targeting:
+      min: 3
+      elite: 5
+    Signature Analysis:
+      elite: 5
```

Comment:
min should be 5 as cannot fly cruiser logi without it, directly affects lock speed of all ships, important for nestor, and as a pre req for cruiser should apply to nestor directly as well imo.